README.md

```diff
@@ -99,6 +99,7 @@ packages:
 |Snowflake|✅|✅|✅ </br>(not nested fields)|✅ </br>(not nested fields)|✅|
 |DuckDB|✅|✅|✅ </br>(not nested fields)|✅ </br>(not nested fields)|✅|
 |Databricks|✅|✅|✅ </br>(not nested fields)|✅ </br>(not nested fields)|✅|
+|Redshift|✅|✅</br> (Include/Exclude Fields Not Supported)|✅ </br>(not nested fields)|✅ </br>(not nested fields)|✅|
```

Comment:
Can you confirm the `__unique__` function returning the window function is compatible with Redshift at this point ? :)